kvmagent/kvmagent/plugins/tensorfusion/container_executor.py

```
@@ -0,0 +1,535 @@
+
+    @classmethod
+    def is_alive(cls, worker):
+        # type: (Worker) -> bool
+        """Check if a worker container is still running."""
+        container_name = getattr(worker, 'container_name', None)
+        if not container_name:
+            return False
+        try:
+            out = cls._docker_class(['inspect', '--format', '{{.State.Running}}', container_name])
+            return out.strip() == 'true'
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)